src/main/java/teammates/ui/controller/AdminEmailLogPageAction.java

```diff
@@ -23,7 +25,12 @@
     private boolean includeAppLogs = true;
     private static final int LOGS_PER_PAGE = 50;
     private static final int MAX_LOGSEARCH_LIMIT = 15000;
-
+    
+    /**
+     * 6 default versions for query, including the current version and its 5 preceding versions.
+     */
+    private static final int MAX_DEFAULT_VERSION_NUMBER = 6;
+    
```

Comment:
Then I would like to resolve #4773 in this PR as well since they are closely related. Is that fine?